nservicebus/azure-service-bus/dlq-forwarding.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Azure Service Bus DLQ Forwarding
+versions: '[7,)'
+reviewed: 2016-08-29
+tags:
+- Azure
+- Cloud
+- Error Handling
+---
+
+## Dead Letter Queue Forwarding
+
+When Azure Service Bus service detects that an endpoint cannot process messages, or when it has been configured not to discard expired or filtered messages, then it will move those messages to a dead letter queue associated with the entity, originally contained those messages.
+
+From a management perspective, this is less than ideal when operations staff has to monitor a multitude of entities for dead lettered or discarded messages.
+
+A better approach might be to consolidate all those messages into a single centralized queue by enabling forwarding on the dead letter queue of each entity, except for the destination entity itself.
+
+NServiceBus supports this capability; it allows to configure forwarding for dead letter queues when it creates entities on behalf of the application.
```

Comment:
@SimonCropp technically the transport.
@indualagarsamy entire document is for version 7+.